### PR TITLE
Feat: Added Toast Notifications

### PR DESCRIPTION
**Pull request by anamkaa**
### Added loaders for actions 
- Add item to the cart
- Remove Item from the cart
- Move an item from the cart to the wishlist
- Add item to the wishlist
- Remove Item from the wishlist
- Move an item from wishlist to cart
- User sign-in
- User Login
- User Logout

https://user-images.githubusercontent.com/75376715/175788520-bb0ab404-fa08-4682-8aa8-362c5d41c92a.mp4

### CODE REVIEW

File: src/components/card/Productcard.jsx [hunk 1]
```diff
@@ -59,11 +60,12 @@ const Productcard = ({ product }) => {
...
               className="card-btn gh-btn gh-btn-primary"
-              onClick={() =>
+              onClick={() => {
                 isLogged
                   ? cartDispatch({ type: "ADD_TO_CART", payload: product })
-                  : navigate("/login")
-              }
+                  : navigate("/login");
+                toast.success("Added to Cart");
+              }}
```
Comment (ashishkatiyar2404) [inline src/components/card/Productcard.jsx]: you can separately call the function here. There is no problem with this.

**Review by ashishkatiyar2404 — APPROVED** [review]
I couldn't find much to improve because your code functionality is good and it looks clean.
You can merge and work on another compo/pages/functionality.💯

File: src/components/card/Productcard.jsx (hunk 1 above)
Comment (MaruthiPothuganti) [inline src/components/card/Productcard.jsx]: Add to cart toast and Add to wishlist toast appears even when the user isn't logged in.

**Review by MaruthiPothuganti — APPROVED** [review]
Suggested a change. Everything else looks good and has cool UI. PR looks good to merge.

File: src/components/card/Productcard.jsx (hunk 1 above)
Comment (anamkaa) [reply src/components/card/Productcard.jsx]: fixed

Comment (anamkaa) [reply src/components/card/Productcard.jsx]: done

**Pull request merged**